### PR TITLE
Implemented react-helmet to display description

### PR DESCRIPTION
**Pull request by Ilithor**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -23,25 +24,39 @@ axios.defaults.baseURL = '/';
...
+              component={UserPage}
+            />
+          </Switch>
+        </div>
+      </ContextProvider>
+    </Router>
+  );
+  return (
+    <div>
+      <Helmet>
```
Comment (jeffersoneagley) [inline src/App.js]: Can Helmet not just live above Navbar?

File: src/App.js [hunk 2]
```diff
@@ -23,25 +24,39 @@ axios.defaults.baseURL = '/';
...
+        <title>SocMon</title>
+        <meta
+          name='description'
+          content='SocMon is a place to meet and communicate with friends and family. Create posts, then comment and like on what others have posted.'
+        />
```
Comment (jeffersoneagley) [inline src/App.js]: You should put helmet in a separate component, and also, you can useLocation to get what page you're on and update the title to be SocMon - whatever page title

File: src/App.js (hunk 1 above)
Comment (Ilithor) [reply src/App.js]: That's what I tried at first, but for whatever reason, the description wouldn't show up unless it was setup this way. Go figure.

File: src/App.js (hunk 2 above)
Comment (Ilithor) [reply src/App.js]: I'll attempt to do that at a later date. At least at this time, that's a pretty low priority compared to the rest of the actions in the project board.

File: src/App.js (hunk 1 above)
Comment (Ilithor) [reply src/App.js]: Decided to convert it over into a context

File: src/App.js (hunk 2 above)
Comment (Ilithor) [reply src/App.js]: fixed

File: src/util/helmetContext.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+            content='SocMon is a place to meet and communicate with friends and family. Create posts, then comment and like on what others have posted.'
+          />
+        </Helmet>
+        {children}
+      </div>
+    </helmetContext.Provider>
+  );
+};
+
+export const useHelmetData = () => {
```
Comment (jeffersoneagley) [inline src/util/helmetContext.js]: probably could just name this useHelmet

**Pull request merged**